### PR TITLE
Lib elektra: rename elektraBackend* to backend*

### PR DESCRIPTION
**Pull request by KurtMi**

### CODE REVIEW

File: doc/TESTING.md [hunk 1]
```diff
@@ -96,6 +96,18 @@ time these tests will fail.
...
+
+According to `src/libs/elektra/libelektra-symbols.map`, all functions starting with:
+
+* libelektra
+* elektra
+* kdb
+* key
+* ks
+
+do not get exported. Test internal functionality by including the corresponding c file.
```
Comment (markus2330) [inline doc/TESTING.md]: It is the other way round: only these functions get exported.

**Comment by markus2330** [comment]
Specific order also here ;(

You can also put these two PRs together, I looked at them already.

**Comment by KurtMi** [comment]
This one first

**Comment by markus2330** [comment]
Thanks!

**Pull request merged**